### PR TITLE
Bump geopandas version to match cuspatial requirements.

### PR DESCRIPTION
**Pull request by thomcom**
Geopandas is 0.9.0+ in cuspatial, now.

### CODE REVIEW

File: conda/recipes/versions.yaml [hunk 1]
```diff
@@ -71,7 +71,7 @@ fsspec_version:
 gdal_version:
   - '>=3.2.0,<3.3.0a0'
 geopandas_version:
-  - '>=0.6, <=0.8.1'
+  - '>=0.9.0'
```
Comment (kkraus14) [inline conda/recipes/versions.yaml:74]: I believe in cuspatial we pinned it to `>=0.9.0,<0.10.0a0` since geopandas is not yet API stable.

Comment (thomcom) [reply conda/recipes/versions.yaml:74]: Thanks for pointing this out. In `cuspatial/meta.yaml` we're using as you specified, in `environments/cuspatial_dev_cuda*.yml` we're using what I specified. This is a discrepancy I've been seeing lately - shouldn't the conda dev environments use the same recipe as the conda release environment from `meta.yaml`? I'd like to get rid of the `.yml` files if possible.

Comment (thomcom) [reply conda/recipes/versions.yaml]: Also, fixed as you requested.

**Review by kkraus14 — APPROVED** [review]

**Comment by thomcom** [comment]
I don't see why the checks are not successful. Console logs don't reveal anything. @ajschmidt8 ?

**Comment by ajschmidt8** [comment]
looks like the failures are related to the calver transition. specifically, we were still waiting on some Docker images and the xgboost package. the xgboost package is available now and the Docker images are currently building. once the Docker images are published, I'll rerun CI here and hopefully we'll get some passing checks!

**Comment by thomcom** [comment]
rerun tests

**Comment by ajschmidt8** [comment]
rerun tests

**Review by ajschmidt8 — APPROVED** [review]

**Pull request merged**